### PR TITLE
Exposes endpoint for deleting applied performance indicators.

### PR DESCRIPTION
**Pull request by patrickberkeley**

### CODE REVIEW

**Comment by patrickberkeley** [comment]
@beerlington will you review this when you have a moment please?

File: app/controllers/v1/applied_performance_indicators_controller.rb [hunk 1]
```diff
@@ -34,6 +34,17 @@ def update
     end
   end
 
+  #TODO Scope to people who are on the project
```
Comment (beerlington) [inline app/controllers/v1/applied_performance_indicators_controller.rb:37]: Now that we have the ability to scope to the current user's projects, why not do it in this PR? I think this would suffice;

``` ruby
# ...
indicator = AppliedPerformanceIndicator.find_by(id: params[:id], project: current_user.projects)
# ...
```

File: app/controllers/v1/applied_performance_indicators_controller.rb [hunk 2]
```diff
@@ -34,6 +34,17 @@ def update
...
+  def destroy
+    indicator = AppliedPerformanceIndicator.find(params[:id])
+
+    if indicator.destroy
+      head :no_content
```
Comment (beerlington) [inline app/controllers/v1/applied_performance_indicators_controller.rb:42]: :+1: I need to start using this

File: app/controllers/v1/applied_performance_indicators_controller.rb (hunk 1 above)
Comment (patrickberkeley) [inline app/controllers/v1/applied_performance_indicators_controller.rb:37]: I was planning to do scoping in a separate PR: https://github.com/AgilionApps/LaunchPad-API/compare/remove-app-perf-indic...scope-applied-perfs

My approach there is slightly different from what you have. Okay to keep them separate?

**Comment by beerlington** [comment]
LGTM
